src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs

```diff
@@ -154,6 +157,13 @@ public static MeterProviderBuilder AddOtlpExporter(this MeterProviderBuilder bui
                 metricExporter = configureExporterInstance(metricExporter);
             }
 
+            var persistentStorage = serviceProvider.GetService(typeof(PersistentBlobProvider));
```

Comment:
@mic-max Lets start with trace exporter since you are updating the tracing example above?